### DISCUSSION TITLE
Advanced Chat Bubble Skeleton - More like WhatsApp

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#3326] [Show and tell]
I wanted some components inside a chat bubble. In case someone needs something similar, you can use hidden class to show and hide specific sections of the chatbubble. I am using javascript to show & hide particular items in the chat bubble.

1. Show reply text with a border with hover underline
2. Show document
3. Show image
4. Show video
5. Show reply button (top right) on hover
6. Show reaction (heart bottom right)

https://play.tailwindcss.com/cpt6kdpvcy

![image](https://github.com/user-attachments/assets/2c13a8ce-209a-4e56-8cd0-23741c924915)

**Comment by saadeghi** [comment]
Thanks for sharing @Mashiane
